src/Controls/samples/Controls.Sample.Droid/Controls.Sample.Droid.csproj

```diff
@@ -79,10 +79,18 @@
     <AndroidResource Include="Resources\values\styles.xml" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
-    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" />
-    <PackageReference Include="Xamarin.Google.Android.Material" />
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection">
+      <Version>$(_MicrosoftDependencyInjectionVersion)</Version>
+    </PackageReference>
+    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData">
+      <Version>2.3.0.1-$(_AndroidXVersion)</Version>
+    </PackageReference>
+    <PackageReference Include="Xamarin.Google.Android.Material">
+      <Version>1.3.0.1-$(_AndroidXVersion)</Version>
+    </PackageReference>
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4">
+      <Version>1.0.0.7-$(_AndroidXVersion)</Version>
+    </PackageReference>
```

Comment:
This doesn't seem right, did an IDE do this?

We should have all `<PackageReference/>` omit their versions so the versions are defined in a single place, such as:
* https://github.com/dotnet/maui/blob/main/eng/AndroidX.targets
* https://github.com/dotnet/maui/blob/main/eng/Microsoft.Extensions.targets

Reply:
Yes VSMAc did that, I didnt realized I missed a few